consensus/src/chained_bft/persistent_storage.rs

```diff
@@ -45,6 +48,57 @@ pub trait PersistentStorage<T>: Send + Sync {
     fn save_highest_timeout_cert(&self, highest_timeout_cert: TimeoutCertificate) -> Result<()>;
 }
 
+pub struct RemoteRecoveryData<T> {
+    epoch: u64,
+    validators: Arc<ValidatorVerifier>,
+    validator_keys: ValidatorSet,
+    genesis_root: Option<(Block<T>, QuorumCert, QuorumCert)>,
```

Comment:
Do you think it's a good opportunity to define some struct like RootInfo? It's not must, but if we're going to pass this triple around, then having RootInfo with specific field names (root_block, root_qc, root_ledger_info) makes it easier to read.

Reply:
Yeah I think it makes sense to have a dedicated struct